chains/dreyerx/metadata.yaml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiKey: abc
+    apiUrl: https://scan.dreyerx.com/api
+    family: blockscout
+    name: DreyerX Scan
+    url: https://scan.dreyerx.com
```

api key not necessary
```suggestion
  - apiUrl: https://scan.dreyerx.com/api
    family: blockscout
    name: DreyerX Scan
    url: https://scan.dreyerx.com
```

---

I've deleted apiKey